app/controllers/projects_controller.rb

```diff
@@ -3,6 +3,7 @@ class ProjectsController < ApplicationController
 
   def index
     @projects = Project.all
+    @projects = @projects.displayable unless params.fetch(:show_hidden, :false) == 'true'
```

Comment:
hiding is by default and then you can force to see them just in case you want to undo it?